RELEASING.md

```diff
@@ -0,0 +1,99 @@
+# Release process
+
+Only for Maintainers.
+
+1.Tag with version to be released e.g.:
```

Comment:
is space lacking intentionally? If list got restarted with the spaces - you can either remove newlines between item and code example or just change to the checkbox style 

``` markdown
- [ ] Blah
```

It should be rendered to the chackbox